podigee/job/__main__.py

```diff
@@ -251,6 +331,7 @@ def transform_podigee_analytics_to_metrics(analytics_data):
 
 for episode in episodes:
     print(episode)
+    episode_published_at = episode.get("published_at", "").split("T")[0]
```

Comment:
Surely there's a better way to get the day from an ISO timestamp?

Reply:
yes, but then we also have to handle the whole timezone issues. with this we just accept the timezone of the hoster